internal/services/admin/server.go

```diff
@@ -204,6 +207,69 @@ func (a *AdminServiceImpl) TriggerWorkflow(ctx context.Context, req *contracts.T
 	}, nil
 }
 
+func (a *AdminServiceImpl) BulkTriggerWorkflow(ctx context.Context, req *contracts.BulkTriggerWorkflowRequest) (*contracts.BulkTriggerWorkflowResponse, error) {
+	tenant := ctx.Value("tenant").(*dbsqlc.Tenant)
+	tenantId := sqlchelpers.UUIDToStr(tenant.ID)
+
+	createContext, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+	var opts []*repository.CreateWorkflowRunOpts
+	var existingWorkflows []string
+
+	if len(req.Workflows) == 0 {
+		return &contracts.BulkTriggerWorkflowResponse{}, status.Error(codes.InvalidArgument, "no workflows provided")
+	}
+
+	if len(req.Workflows) > 1000 {
+		return nil, status.Error(codes.InvalidArgument, "maximum of 1000 workflows can be triggered at once")
+	}
+
+	opts, existingWorkflows, err := getOpts(ctx, req.Workflows, a)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(opts) == 0 {
+		if len(existingWorkflows) == 0 {
+			return &contracts.BulkTriggerWorkflowResponse{}, status.Error(codes.InvalidArgument, "no suitable new workflows provided")
```

Comment:
```suggestion
			return nil, status.Error(codes.InvalidArgument, "no suitable new workflows provided")
```

nit, why are we returning this?